src/app/container/search/search.component.html

```diff
@@ -0,0 +1,7 @@
+<div class="ekart--search--product">
+    <input class="ekart-search-product-input" #searchInput>
+    <button class="btn btn-search" (click)="updateSearchText()">Search</button>
+</div>
+<div class="ekart-search-result-for">
+    <p *ngIf="searchText.trim() != ''"><strong>Search result for:</strong> {{ searchText }}</p>
```

Comment:
1. It is a bad practice to call a method inside the template because of performance reasons
2. you should always use !== instead of !=